### ISSUE TITLE
Filter by Roles and Responsibilities

### ISSUE THREAD

**Issue by poorgeek** [w3c/wai-wcag-quickref#108]
It would be nice to filter and share Success Criteria using WAI-Engage's [Accessibility Responsibility Breakdown](http://www.w3.org/community/wai-engage/wiki/Accessibility_Responsibility_Breakdown) table. The benfit being that you could share a link with someone based on their assigned role within them team of success criteria that they are responsbile for.

**Comment by yatil** [comment]
Hi @poorgeek,

thanks for your feedback.
We have indeed discussed this several times, but it created quite a challenge in actually implementing it. we are currently updating the tags and hope that those work better universally. We found it hard to implement responsibilities in a sensible way if users have overlapping responsibilities, which is often the case. Also many success criteria are written in a way (for reasons) so they work for many roles and responsibilities. 

I will keep this issue open as a reminder, maybe we can add such a functionality in a future version of the tool.
